File: CPP/product.cpp
#include <iostream>
using namespace std ;

#include "product.h"

Product::Product (int ID, int _harga) {
	productID = ID;
	harga = _harga;
}

int Product::getProductID(){
	return productID;
}

int Product::getHarga(){
	return harga;
}

File: GUI-nya/Yg Package/JavaGUI/src/products/Product.java
package products;

public class Product{
	private int productID;
	private int harga;
	
	public Product(int _productID, int _harga){
		productID = _productID;
		harga = _harga;
	}
	
	public int getProductID(){
		return productID;
	}
	
	public int getHarga(){
		return harga;
	}
}
